scripts/benchmark_all.sh

```diff
@@ -74,17 +84,14 @@ do
   for benchmark in $benchmarks
   do
     echo "Running $benchmark for $commit... (single-threaded)"
-    ( "${build_folder}"/"$benchmark" -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.log"
+    ( "${build_folder}"/"$benchmark" -r ${runs} -w ${warmup_seconds} -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st.log"
 
     if [ "$benchmark" = "hyriseBenchmarkTPCH" ]; then
       echo "Running $benchmark for $commit... (single-threaded, SF 0.01)"
-      ( "${build_folder}"/"$benchmark" -s .01 -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st_s01.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st_s01.log"
+      ( "${build_folder}"/"$benchmark" -s .01 -r ${runs} -w ${warmup_seconds} -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st_s01.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st_s01.log"
 
-      echo "Running $benchmark for $commit... (single-threaded, SF 1.0)"
-      ( "${build_folder}"/"$benchmark" -s 1 -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st_s1.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_st_s1.log"
-
-      echo "Running $benchmark for $commit... (multi-threaded, clustered for pruning)"
-      ( "${build_folder}"/"$benchmark" --scheduler --clients ${num_mt_clients} -s 10 --dont_cache_binary_tables --clustering=Pruning -o "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt_clustered.json" 2>&1 ) | tee "${build_folder}/benchmark_all_results/${benchmark}_${commit}_mt_clustered.log"
+      echo "Running $benchmark for $commit... (multi-threaded, shuffled)"
```

Comment:
I am wondering if we should include a non-shuffled run. We are currently pretty bad when it comes to "many cores, single client". This benchmark would allow us to keep an eye on that.

Reply:
Sure. I think we can afford ~20 minutes more.

Reply:
Done